app-emulation/aranym/aranym-1.1.0.recipe

```diff
@@ -5,12 +5,12 @@ HOMEPAGE="https://aranym.github.io/"
 COPYRIGHT="2001-2014 ARAnyM developer team"
 LICENSE="GNU GPL v2"
 REVISION="1"
-SOURCE_URI="https://github.com/aranym/aranym/archive/ARANYM_1_0_2.tar.gz"
-CHECKSUM_SHA256="c0935aee14cca55c08e75a94b77dcd69f326ae563ae371b22c7b9d8efd82e24e"
+SOURCE_URI="https://github.com/aranym/aranym/archive/ARANYM_${portVersion//./_}.tar.gz"
+CHECKSUM_SHA256="1636cf9ec68bd2089f537e3b5e82d04fafec61a4f8fb5e216a263d357e935b63"
 SOURCE_FILENAME="anarym-$portVersion.tar.gz"
-SOURCE_DIR="aranym-ARANYM_1_0_2"
+SOURCE_DIR="aranym-ARANYM_${portVersion//./_}"
 
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
As there is no x86 primary architecture we should see to remove it from the recipes:
```diff
- ARCHITECTURES="x86_gcc2 x86 x86_64"
+ ARCHITECTURES="x86_gcc2 x86_64"
```